.github/workflows/integration.yml

```diff
@@ -149,15 +149,15 @@ jobs:
         contents: write
     steps:
 
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           # fetch all branches
           fetch-depth: 0
 
       # Configure git user for later command induced commits
       - uses: fregante/setup-git-user@v1
 
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
           node-version: 18
```

Comment:
I think you forgot about this one ?

Reply:
Talked about it IRL, it was just missed.

Reply:
I think we should redo the publication test on your fork/npm account with this modification just to be sure since it is in the `publish` action. WDYT?

Reply:
Thanks :)